### DISCUSSION TITLE
globals being overwrite by Express.js

### DISCUSSION THREAD

**Discussion by YahangWu** [harttle/liquidjs#532] [Working with Express]
Hi,

I have an object called `settings` in `globals`, and when I call `res.render(parentTemplate);`, the `settings` is being overwrite by Express.js. 

In the child template that is used in `parentTemplate` with `{% render childTemplate.liquid %}`, the globals `settings` can be accessed. 

Is there a way that I can prevent this to happen?

**Comment by harttle** [comment]
Do you have a minimal repro snippet, or project? So I can help you debug what's actually happenning.

**Reply by YahangWu** [reply]
Hi,

Sorry missed your message! I haven't got the chance to get one minimal one ready but will be in the following days. Will get back to you soon. Thanks!

**Reply by YahangWu** [reply]
I found the issue was on liquidjs 9.25 and after upgrading to the latest this was not an issue anymore. Sorry for the confusion.

**Reply by harttle** [reply]
Thank you for the information!

**Reply by YahangWu** [reply]
Sorry I lied, it is still having the same problem. I will create a demo package to show the issue.  😔
